initialization/initialization_test.go

```diff
@@ -30,15 +29,21 @@ var (
 	commitmentAtxId = make([]byte, 32)
 )
 
-func TestInitialize(t *testing.T) {
+func getTestConfig(tb testing.TB) (config.Config, config.InitOpts) {
 	cfg := config.DefaultConfig()
-	cfg.LabelsPerUnit = 1 << 12
 
 	opts := config.DefaultInitOpts()
-	opts.DataDir = t.TempDir()
+	opts.Scrypt.N = 16 // speed up initialization
```

Comment:
This could actually be the smallest possible value of 2 for best test speed
```suggestion
	opts.Scrypt.N = 2 // speed up initialization
```

💡 IIRC, there was a test that didn't want to work with N=2 because... init was too fast (a test checking initializer state transitions or something like that).